patches/minecraft/net/minecraft/client/renderer/LightTexture.java.patch

```diff
@@ -0,0 +1,19 @@
+--- a/net/minecraft/client/renderer/LightTexture.java
++++ b/net/minecraft/client/renderer/LightTexture.java
+@@ -110,7 +110,16 @@
+                      f11 = 0.28F + f8 * 0.75F;
+                      f12 = 0.25F + f9 * 0.75F;
+                   }
++                  
++                  float[] colors = {f10, f11, f12};
++                  world.func_201675_m().getLightmapColors(p_205106_1_, f, f4, f5, colors);
++                  f10 = colors[0]; f11 = colors[1]; f10 = colors[2];
```

Comment:
Minor typo here - second `f10` should be `f12`

Reply:
Darn, and I thought I caught them all.